tests/common/secure_sockets/aws_test_tcp.c

```diff
@@ -1892,6 +1892,9 @@ static void prvConnect_InvalidAddressLength( Server_t xConn,
     {
         if( xConn == eNonsecure )
         {
+            xEchoServerAddress.ucLength = ( uint8_t ) sizeof( SocketsSockaddr_t );
```

Comment:
Should this init code go outside of this if statement?

Reply:
My bad. Will move. Thanks.